WebManifest/mani-tests.ts

```diff
@@ -52,6 +52,25 @@ export const checkIcons = (data) => {
   }
 }
 
+export const checkMaskableIcon = (data) => {
+  const hasIcons = checkIcons(data);
+  
+  if (hasIcons) {
+    let hasMask = false;
+
+    data.icons.forEach((icon) => {
+      if (icon.purpose && icon.purpose.includes('maskable')) {
+        hasMask = true;
+      }
+    })
```

Comment:
if we use a for loop here we could return out of the loop if we find a maskable, which will save us primarily a few nanoseconds, but maybe a millisecond if the list is large, lol. Not a big deal, but thought I'd mention.